src/javascript/app/pages/user/new_account/digital_options.js

```diff
@@ -0,0 +1,53 @@
+const BinarySocket     = require('../../../base/socket');
+const Client           = require('../../../base/client');
+const getElementById   = require('../../../../_common/common_functions').getElementById;
+const Url              = require('../../../../_common/url');
+const ClientBase       = require('../../../../_common/base/client_base');
+const showLoadingImage = require('../../../../_common/utility').showLoadingImage;
+const State            = require('../../../../_common/storage').State;
+
+const DigitalOptions = (() => {
+
+    let is_virtual,
+        el_welcome_container,
+        upgrade_info,
+        is_uk,
+        is_unwelcome_uk;
+
+    const init = () => {
+        is_virtual            = Client.get('is_virtual');
+        upgrade_info          = ClientBase.getBasicUpgradeInfo();
+        el_welcome_container  = getElementById('welcome_container');
+        is_uk                 = Boolean(Client.get('residence') === 'gb');
```

Comment:
I believe `===` is already a boolean converter and it doesn't need another `Boolean` wrapper around it, no?

Reply:
Done